geronimo-iia/async-btree
# Dev 1.3.0

Feature:
- add default parallel asyncio implementation if curio is not present.
- add run_once decorator
- bunp pytest dependency
- Deprecation notice on `async_btree.utils.run` 
- Add BTreeRunner context manager to drive multiple independant btree execution.
  This implementation works with curio and asyncio.
  `asyncio` support only for python >= 3.11.
- remove `__version__` package attribute. Single source of true is pyproject.toml and git.
